web-registry/src/components/molecules/MetaDetail/MetaDetail.BaseValue.tsx

```diff
@@ -7,34 +9,49 @@ import { formatDate, formatNumber } from 'web-components/lib/utils/format';
 import { LinkWithArrow } from 'components/atoms';
 
 import { BaseValue, isCompactedNameUrlOrOptionalUrl } from './MetaDetail.types';
+import { fromISO8601 } from './MetaDetail.utils';
 
 export type Props = {
   value?: BaseValue;
   rdfType?: ExpandedTermDefinition['@type'];
   bodySize?: TextSize;
+  sx?: SxProps<Theme>;
 };
 
-const MetaDetailBaseValue: React.FC<Props> = ({ value, rdfType, bodySize }) => {
-  if (!value) return null;
-
+const MetaDetailBaseValue: React.FC<Props> = ({
+  value,
+  rdfType,
+  bodySize,
+  sx,
+}) => {
   let formattedValue: string | undefined;
   const isNumber = typeof value === 'number';
   const isString = typeof value === 'string';
+  const isBoolean = typeof value === 'boolean';
   if (isString || isNumber) {
     if (isNumber) {
       formattedValue = formatNumber({ num: value });
     } else if (rdfType === 'xsd:date') {
       formattedValue = formatDate(value);
+    } else if (rdfType?.includes('Duration')) {
```

Comment:
currently just "Duration" inclusion checking this but ultimately once this: https://github.com/regen-network/regen-registry-standards/pull/57#discussion_r1273270388 is fixed, we should check for `schema:Duration` equality